tests/test_config.py

```diff
@@ -427,45 +458,48 @@ def test_user_defaults(tmp_path_factory, user_defaults, data, expected):
             },
             dedent(
                 """\
-            A string: yosemite
-            """
+                A string: yosemite
+                """
             ),
             dedent(
                 """\
-            A string: red rocks
-            """
+                A string: red rocks
+                """
             ),
         ),
     ],
 )
 def test_user_defaults_updated(
-    tmp_path_factory,
-    user_defaults_initial,
-    user_defaults_updated,
-    data_initial,
-    data_updated,
-    expected_initial,
-    expected_updated,
-):
+    tmp_path_factory: pytest.TempPathFactory,
+    user_defaults_initial: AnyByStrDict,
+    user_defaults_updated: AnyByStrDict,
+    data_initial: AnyByStrDict,
+    data_updated: AnyByStrDict,
+    expected_initial: str,
+    expected_updated: str,
+) -> None:
     src, dst = map(tmp_path_factory.mktemp, ("src", "dst"))
     with local.cwd(src):
         build_file_tree(
             {
-                src
-                / "copier.yaml": """\
+                (src / "copier.yaml"): (
```

Comment:
Is it intended that the file path contains `src / ...` here while the current working directory is `src` already due to  `with local.cwd(src):`?

Reply:
probably just an overlook

Reply:
So I'll remove the `src / ` part, it shouldn't change the behavior of that test.